central/notifiers/email/email.go

```diff
@@ -256,6 +256,8 @@ func (m Message) writeContentBytes(buf *bytes.Buffer) {
 
 	writer := multipart.NewWriter(buf)
 	boundary := writer.Boundary()
+	var idx int
+	numberOfAttachments := len(m.Attachments)
 
 	if m.EmbedLogo {
 		buf.WriteString(fmt.Sprintf("Content-Type: multipart/mixed; boundary=\"%s\"\r\n", boundary))
```

Comment:
After setting content type and adding boundary using `bool mixedType` as per previous comment, you can remove this and line below.